src/main/java/core/basesyntax/model/User.java

```diff
@@ -49,13 +49,14 @@ public boolean equals(Object o) {
             return false;
         }
         User user = (User) o;
-        return Objects.equals(login, user.login)
+        return Objects.equals(id, user.id)
```

Comment:
same

Reply:
I don't think that `new User(0, 'admin', 'password', 20)` should be equal to `new User(1, 'admin', 'password', 20)` - they have different IDs. What do you think?